tests/baselines/reference/aliasDoesNotDuplicateSignatures.symbols

```diff
@@ -8,7 +8,7 @@ declare namespace demoNS {
 >f : Symbol(f, Decl(demo.d.ts, 0, 26))
 }
 declare module 'demoModule' {
->'demoModule' : Symbol("demoModule", Decl(demo.d.ts, 2, 1))
+>'demoModule' : Symbol('demoModule', Decl(demo.d.ts, 2, 1))

```

Comment:
I'm surprised to see symbol baselines change; were we inconsistent with these before?

Reply:
Symbol printing is involved in type printing and error printing. I applied the 'preserve source code delimiters in everything except errors' to the symbol printing too. This does mean that the delimiters are now preserved in symbol baselines where they previously were not.

Were they always replaced before? Don't think so. Some error messages  used symbol printing for the property names and we can see in error baselines changes that some of those kept the `'` delimiter instead of having it replaced with `"`